app/views/games/index.html.erb

```diff
@@ -1,11 +1,30 @@
 <div class="booyah-box col-xs-10 col-xs-offset-1">
-<h1 class="text-center">List of Games</h1>
+<h3 class="text-center"><%= current_user.email %></h3>
+<% if @my_games.present? %>
+  <h3>My games</h3>
+  <% @my_games.each do |game|%>
+  <%= link_to game.name, '#' %>
+  <% end %>
+ 
+
+<% end %>
+<% if @games.present? %>
+	<h1 class="text-center">Available Games</h1>
+<% end %>

```

Comment:
The `@games.present?` conditional will only show or hide the title.  Shouldn't it display a message to the user if no games are available, such as "No games available.  Create one?".